.travis.yml

```diff
@@ -3,5 +3,5 @@ rust:
     - nightly-2016-02-11
 
 script:
-    - cargo build # Linter is also executed here
+    - cargo test
     - RUST_BACKTRACE=1 cargo test
```

Comment:
Ups, I though you were only exporting an environment variable. Then, let's remove `cargo build` at all as `cargo test` implies to build. Ok?